cmd/infra/powervs/create.go

```diff
@@ -314,9 +314,8 @@ func getIAMAuth() *core.IamAuthenticator {
 	}
 }
 
-// setupBaseDomain get domain id and crn of given base domain
-// TODO(dharaneeshvrd): Currently, resource group provided will be considered only for VPC and PowerVS. Need to look at utilising a common resource group in future for CIS service too and use it while filtering the list
-func (infra *Infra) setupBaseDomain(options *CreateInfraOptions) (err error) {
+// getCisDomainDetails getting CIS domain details like CRN and domainID
+func getCisDomainDetails(baseDomain string) (CisCRN string, CisDomainID string, err error) {
```

Comment:
is there a reason to use by named return variables? it's not consistent with the rest of the code base of the project and IMO makes readability less obvious than explicit returns.

Reply:
Yes I can see on rest of the code base not using named return variables. Will refactor the code to follow the same.